UI/window-basic-preview.cpp

```diff
@@ -658,9 +658,48 @@ void OBSBasicPreview::UpdateCursor(uint32_t &flags)
 
 	if (!flags && (cursor().shape() != Qt::OpenHandCursor || !scrollMode))
 		unsetCursor();
-	if (cursor().shape() != Qt::ArrowCursor)
+	if ((cursor().shape() != Qt::ArrowCursor) || flags == 0)
 		return;
 
+	if (flags & ITEM_ROT) {
+		setCursor(Qt::OpenHandCursor);
+		return;
+	}
+
+	float rot = obs_sceneitem_get_rot(stretchItem);
```

Comment:
Rename `rot` to `rotation`, `q` to `quadrant`, and `corner` to `isCorner`. We have the storage, we can write out variable names. 😄